code/modules/vore/fluffstuff/custom_clothes_vr.dm

```diff
@@ -1931,7 +1931,3 @@ Departamental Swimsuits, for general use
 	default_worn_icon = 'icons/vore/custom_clothes_vr.dmi'
 	color = COLOR_NAVY
 	sprite_sheets = null
-
-/obj/item/clothing/under/fluff/excess/get_worn_icon_state(var/slot_name)
-	. = ..()
-	. = copytext(. , 1 , lentext( . ) - 1)  //get rid of the extra "_s" added by /obj/item/clothing/under/get_worn_icon_state()
```

Comment:
-squint- Who in the fuck wrote this?

Reply:
This was s'posed to be a fix for Excess' custom item overlaying wrong going by the PR related. F.